apps/testing-javascript/src/path-to-purchase/pricing.tsx

```diff
@@ -11,14 +11,14 @@ import type {
   FormattedPrice,
 } from '@skillrecordings/commerce-server/dist/@types'
 import {CheckCircleIcon} from '@heroicons/react/outline'
-import {useDebounce} from '@skillrecordings/react'
```

Comment:
the core of this is that dependencies on sass in various packages are causing nextjs in node_modules/pnpm to have two different versions

this makes the build break in mysterious all-day tears sorts of ways